bool is_enemy_recognized
float32 enemy_dist
float32 enemy_direct 
